src/pages/docs/tutorials/tutorial-hello-enact/adding-sandstone-support/index.md

```diff
@@ -0,0 +1,81 @@
+
+On top of this, Sandstone establishes a set of screen types for our supported television resolutions that are configured for the application by ThemeDecorator.
+
+### Internationalization Support
+
+In order to support delivery of applications to multiple locales, Enact integrates with the [iLib](https://sourceforge.net/projects/i18nlib/) internationalization library through our own module, `@enact/i18n`. This module provides the glue between the two code bases to correctly load locale resources and expose the results within an Enact app.
+
+ThemeDecorator uses I18nDecorator, a HoC from the `@enact/i18n` module, to add the appropriate CSS classes for the locale. I18nDecorator is also responsible for passing the `rtl` flag and `updateLocale` method to the app via [React's context](https://reactjs.org/docs/context.html).
+
+### Spotlight Support
+
+Sandstone applications must support 5-way navigation and selection to allow them to be used with only a remote control. The `@enact/spotlight` module provides the logic to support this and it is bootstrapped into a Sandstone app via ThemeDecorator and the SpotlightRootDecorator.
+
+> All Sandstone components that should be navigable via 5-way already include the necessary support
+> out of the box. If you build your own components, you may need to add your own spotlight support
+> using Spottable from `@enact/spotlight/Spottable`.
+
+## Updating App.js
+
+To use a Higher-order Component you've imported, pass another component (or DOM node name) to the HoC, optionally including the configuration object. For ThemeDecorator, you can opt out of internationalization, resolution independence, or spotlight support by setting `i18n`, `ri`, or `spotlight` respectively to `false`. For our sample application, we will use the defaults.
+```js
+import ThemeDecorator from '@enact/sandstone/ThemeDecorator';
+
+const AppBase = kind({
+	// omitted for brevity
+});
+
+const App = ThemeDecorator(AppBase);
+export default App;
+export {
+	App, 
+	AppBase
+};
+```
+> **Note**: We've renamed the stateless component from `App` to `AppBase`. The
+> Enact framework uses this convention to distinguish between stateless components and their wrapped or stateful versions.
+> If you need to use the stateless version of a component, it will be exported as a named export with
+> `Base` appended to the name.
+>
+> You may find this convention useful with your components so that module exports are predictable.
+
+## Conclusion
+
+This section introduced ThemeDecorator, which adds key styling and features for applications targeting the TV. If you have been following along, you now have a basic Sandstone app welcoming you to Enact. You can get this source from our repository [Hello, Enact](https://github.com/enactjs/samples/tree/develop/tutorial-hello-enact).
```

Comment:
Please don't add the source code URL.

Reply:
OK.
The source code URL will be removed.